src/Makefile.test.include

```diff
@@ -44,6 +44,7 @@ FUZZ_TARGETS = \
   test/fuzz/eval_script \
   test/fuzz/fee_rate \
   test/fuzz/fee_rate_deserialize \
+  test/fuzz/fees \
```

Comment:
18736 why is this added here? did we skip a backport?

Reply:
The original `fees.cpp` was introduced in [#18565](https://github.com/bitcoin/bitcoin/pull/18565) through 44fb2a596b4a1aa70253c4145c35be6de68da22a which addressed logic that Dash doesn't have. `StringForFeeReason` exists in Dash and so we created the file on a different commit as opposed to the original file history ([source](https://github.com/practicalswift/bitcoin/commits/a4e3d13df6a6f48974f541de0b5b061e8078ba9a/src/test/fuzz/fees.cpp))